automatic/zotero/tools/chocolateyInstall.ps1

```diff
@@ -1,16 +1,20 @@
 ﻿$ErrorActionPreference = 'Stop'
 
-$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$toolsPath = Split-Path -parent $MyInvocation.MyCommand.Definition
+$Installers = Get-ChildItem -Path $toolsPath -Filter '*.exe' |
```

Comment:
I'm not sure I understand why, but easy enough.

Reply:
Two reasons generally:

1. This way we know exactly which installer is being used, and prevent any installers we do not want to use from being collected.
2. There have been a history of the wrong installer being used, especially during upgrade or even multiple installers found (again mostly for upgrades) which causes the package to fail installation.
   Not sure if it is still the case, but in the past old installers would not be removed from the package during an upgrade (and sometimes even when we manually deleted it, it wouldn't be deleted), this caused packages using a glob pattern to fail the entire installation because of it.